lib/index.js

```diff
@@ -115,24 +115,56 @@ $(document).ready(function(){
     });
   }
 
+  function postFavorites(cityState) {
+    $.post(`https://sweather-weather.herokuapp.com/api/v1/favorites?location=${cityState}&api_key=71caee34584ad1b2fcc7`, function(data, status) {
```

Comment:
If you add the var cityState inside the function, you won't need to pass it in, but that's certainly a design decision.